src/KeyVault/KeyVault/Properties/Resources.resx

```diff
@@ -483,4 +483,7 @@ You can find the object ID using Azure Active Directory Module for Windows Power
   <data name="VaultDoesNotExist" xml:space="preserve">
     <value>Vault '{0}' does not exist in current subscription.  If this vault exists in your tenant, please switch to the correct subscription in order to modifiy the Access Policies of this vault.</value>
   </data>
+  <data name="ActiveDirectoryClientNull" xml:space="preserve">
+    <value>ActiveDirectoryAccessToken is not available. Please either use the ObjectId parameter set or authenticate with an ActiveDirectoryAccessToken.</value>
```

Comment:
This is not necessarily related to token auth - this could be because a ServicePrincipal hasn't been granted access to ActiveDirectory.  We should say somethign like:  

'The current credentials do not have acess to Azure Active Directory.  Please either use an ObjectId to refer to ActiveDirectory objects, or log in using credentials that have access to Azure Active Directory'